docs/modules/operation/pages/deep-dive/performance-data-collection/resource-types.adoc

```diff
@@ -141,12 +167,19 @@ Replace with the following:
 
 === SiblingColumnStorageStrategy
 
-[options="header, autowidth"]
+[options="header" "autowidth", cols="1,2"]
 |===
-| Parameter           | Description
-| sibling-column-name | Alternate string value to use for index.
-| replace-first       | Regex pattern, replaces only the first match.
-| replace-all         | Regex pattern, replaces all matches.
+| Parameter
+| Description
+
+| sibling-column-name
+| Alternate string value to use for index.
```

Comment:
Name of another `mibObj` in the same `group`, whose type must be `string`. This object's value will form the starting point for the index.

Reply:
```suggestion
| The name of another `mibObj` in the same group.
This string value forms the starting point for the index.
```